deno/README.md

```diff
@@ -0,0 +1,59 @@
+# Flatpak Deno Generator
+
+run from jsr
+
+```
+deno -RN -W=. jsr:@flatpak/flatpak-deno-generator deno.lock
+```
+
+or locally from this repo
+
+```
+deno -RN -W=. main.ts deno.lock
+```
+
+This will create a `deno-sources.json` that can be used in flatpak build files:
+
+- it creates and populates `./deno_dir` with npm dependencies
+- it creates and populates `./vendor` with jsr + http dependencies
```

Comment:
are these created by the script? But the script will be run externally by the maintainer to update `deno-sources.json`, these vendor directories won't be in the build.

because later you have `- mv ./vendor src/`

Reply:
no they are not created by the script , I just simply meant that this is the 2 folders that the sources file will create, basicly dest here https://github.com/flathub/io.github.sigmasd.VirtAudio/blob/master/deno-sources.json#L6

vendor need to be next to deno.json, it just happens in tht example, deno.json is located under src

Reply:
I improved the docs